### ISSUE TITLE
No New Release Since 1 Year 

### ISSUE THREAD

**Issue by gauravdua27** [firebase/FirebaseUI-Android#2137]
This Repo is an Official Firebase UI library and it has not released any update for 1 year for SafetyNet and many more things.  

Is this Repo Planning for releases in Future, Or do we consider this Repo as Deprecated and Not maintained now?

**Comment by rocketraman** [comment]
:+1: The Google Play Store is now requiring targeting SDK 34. This Google library is a blocker.

**Comment by JoniGoos** [comment]
Any update on this one?

**Comment by ChrisKruegerDev** [comment]
@JoniGoos I guess there is no further interest in updating this library 😔

**Comment by ttermeer-reboundcg** [comment]
@thatfiredev can we get an official statement?

**Comment by OleksandrGrument** [comment]
@thatfiredev Any upfates?

**Comment by thatfiredev** [comment]
Hey all,

Apologies for the long silence on this one!
We are finally picking FirebaseUI Android back up and we have just released [version 9.0.0](https://github.com/firebase/FirebaseUI-Android/blob/master/docs/upgrade-to-9.0.md) which contains fixes to all the issues mentioned in https://github.com/firebase/FirebaseUI-Android/issues/2137#issuecomment-2022250344 .

Moving forward I'm committed to maintaining this library and address the remaining open issues. The top priorities for now are #2177 and #1917, and you'll see those in the upcoming releases. Any new issues that arise will also be prioritized accordingly.

Thank you so much for your patience!

_I'm closing this issue for now, but feel free to leave a comment for further discussion/clarification._

**Comment by bocops** [comment]
@thatfiredev The Firebase documentation I mentioned in my earlier comment still claims that v7.2.0 should be used.

**Comment by thatfiredev** [comment]
@bocops good call. I just pushed an update and it should be live in a few minutes on https://firebase.google.com/docs/auth/android/firebaseui
